test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/DirectMethodLongHaulReportData.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Modules.Test.TestResultCoordinator.Reports.DirectMethod
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+
+    class DirectMethodLongHaulReportData
+    {
+        public static IEnumerable<object[]> GetCreateReportData =>
+        new List<object[]>
+        {
+            new object[]
```

Comment:
nit: iirc you are just following the existing pattern, but is there a reason why we didn't make some sort of abstraction for this test data?